lib/twitter_ads.rb

```diff
@@ -7,9 +7,11 @@
 #
 require 'oauth'
 require 'json'
+require 'multi_json'
+
+ADS_API_ENDPOINT = 'ads-api.twitter.com'
+TRACE = false
```

Comment:
The purpose was more internal debug but why not allowing it to be modified. We could (and should) update locally what happen on twitter side. Question is at which rate, and when....